CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Changelog
 
+## `v10.2.0`
+
+### Miscellaneous small additions and changes
+
+- **[ci]** improved caching [#2197](https://github.com/docker-mailserver/docker-mailserver/pull/2197)
+- **[ci]** refactored spam tests and introduced common container setup template [#2198](https://github.com/docker-mailserver/docker-mailserver/pull/2198)
+- **[fix]** update Fail2Ban wrapper to propagate errors to user [#2170](https://github.com/docker-mailserver/docker-mailserver/pull/2170)
+- **[fix]** Dockerfile `sed`'s are now checked [#2158](https://github.com/docker-mailserver/docker-mailserver/pull/2158)
+- **[general]** Updated default value of `ONE_DIR` to `1` [#2148](https://github.com/docker-mailserver/docker-mailserver/pull/2148)
+- **[docs]** updated Kubernetes documentation [#2111](https://github.com/docker-mailserver/docker-mailserver/pull/2111)
+- **[docs]** introduced dedicated Podman documentation [#2179](https://github.com/docker-mailserver/docker-mailserver/pull/2179)
+- **[docs]** miscellaneous documentation improvements
+- **[misc]** introduced GitHub issue forms for  issue templates [#2160](https://github.com/docker-mailserver/docker-mailserver/pull/2160)
+- **[misc]** removed old `mkcert.sh` [#2196](https://github.com/docker-mailserver/docker-mailserver/pull/2196)
```

Comment:
Until the last version, there was no need to manually modify mkcert.sh, because there where corresponding 'sed' lines in the Dockerfile. This release removes those lines, however also mkcert.sh was removed. So for long-time users there should be no different.